constraints.txt

```diff
@@ -2,35 +2,36 @@ appdirs==1.4.3
 atomicwrites==1.2.1
 attrs==18.2.0
 bernhard==0.2.6
-black==18.9b0
+black==19.3b0
 certifi==2018.8.24
```

Comment:
We might update this to ```certifi==2019.3.9```, but we can do that some other time as well. Not relevant for the docker image.

Reply:
I will update the PR, "upgrade all dependencies" was a lie